### PR TITLE
Feature/Add CC-BY-NC-ND and CC-BY-SA4-0 [PLAT-722]

### PR DESCRIPTION
**Pull request by pattisdr**
# Purpose
Add the following licenses to OSF license options dropdown:

CC-BY-NC-ND (requested by Earthrxiv)
CC-BY-SA 4.0 (requested by LawArXiv and Thesis Commons user)

# Ticket
https://openscience.atlassian.net/browse/PLAT-722

### CODE REVIEW

**Comment by pattisdr** [comment]
For whoever reviews this, please verify that I've pasted the correct licenses in their entirety.

File: licenses/ccbyncnd-4.0.txt [hunk 1]
```diff
@@ -0,0 +1,95 @@
+Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International
```
Comment (erinspace) [inline licenses/ccbyncnd-4.0.txt]: [A plaintext version of the license I found](https://creativecommons.org/licenses/by-nc-nd/4.0/legalcode.txt) has an extra section at the top, but I dunno if others have this and we don't routinely include them?

File: licenses/ccbysa-4.0.txt [hunk 2]
```diff
@@ -0,0 +1,103 @@
+Creative Commons Attribution-ShareAlike 4.0 International
```
Comment (erinspace) [inline licenses/ccbysa-4.0.txt]: [Ditto!](https://creativecommons.org/licenses/by-sa/4.0/legalcode.txt)

**Review by erinspace — COMMENTED** [review]
Just a question about a section at the top of each license!

File: licenses/ccbyncnd-4.0.txt (hunk 1 above)
Comment (pattisdr) [reply licenses/ccbyncnd-4.0.txt]: Thanks for pointing this out @erinspace, looks like I was including too much.  [The other CC licenses](https://github.com/CenterForOpenScience/list-of-licenses/blob/master/licenses/ccby-4.0.txt) have this top section, but it doesn't look like we include them.  I guess it's not a part of the license, but more clarifying what Creative Commons is.

**Review by caseyrollins — APPROVED** [review]

**Pull request merged**